.github/workflows/e2e.yml

```diff
@@ -1,4 +1,4 @@
-name: dpc-sdp/ripple/nightwatch
+name: dpc-sdp/ripple/e2e
```

Comment:
@tim-yao Do we need to update all input values for test_type from `test_type: ':nightwatch:'` to `test_type: ':e2e:'` as well?

Reply:
> @tim-yao Do we need to update all input values for test_type from `test_type: ':nightwatch:'` to `test_type: ':e2e:'` as well?

That is a good suggestion, please do. Cheers!